src/components/tooltip/tooltip.js

```diff
@@ -129,7 +129,7 @@ function MdTooltipDirective($timeout, $window, $$rAF, $document, $mdUtil, $mdThe
     function bindEvents () {
       var mouseActive = false;
       var enterHandler = function() {
-        if (!hasComputedStyleValue('pointer-events','none')) {
```

Comment:
This change does not make sense. Valid values for the `pointer-events` style are:

*  auto
*  inherit
*  none